Assets/HoloToolkit/Common/Scripts/Singleton.cs

```diff
@@ -62,6 +62,16 @@ public static bool IsInitialized
             }
         }
 
+        /// <summary>
+        /// Awake and OnEnable safe way to check if a Singleton is initialized.
+        /// </summary>
+        /// <returns>The value of the Singleton's IsInitialized property</returns>
+        public static bool ConfirmInitialized()
+        {
+            T access = Instance;
```

Comment:
This wasn't in the master branch already?

Any other singleton changes we didn't have?

Reply:
it wasn't. i will look and open a PR if i see any more missing code

Reply:
I thought the same thing!